### ISSUE TITLE
No documentation available what the enrichers offers me as an user

### ISSUE THREAD

**Issue by DannyvdSluijs** [theseer/phpdox#248]
Working on fine-tuning the phpdox config I can across the enrichers. Some enrichers are self explanatory and some aren't. Even the ones that seems self explanatory do something more than expected, e.g. the git enricher, is was expecting some git tag being mentioned (Which was on the index page). However I wasn't expecting a history of commits per file which was i very nice surprise but would not have guessed it from its name.

**Comment by theseer** [comment]
You are right, phpDox is in quite desperate need of docs and a more descriptive website listing more feature details and features.

I hope to address these issues as soon as possible but am currently quite busy with other things so it may take a while.
